docs/data-sources/vpcep_endpoints.md

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "VPC Endpoint (VPCEP)"
+---
+
+# flexibleengine\_vpcep\_endpoints
+
+Use this data source to get VPC endpoints.
+
+## Example Usage
+
+```hcl
+data "flexibleengine_vpcep_endpoints" "all_services" {
+}
+
+data "flexibleengine_vpcep_endpoints" "dns_service" {
+  endpoint_service_name = "dns"
+}
+```
+
+## Argument Reference
+
+* `endpoint_service_name` - (Optional, String) Specifies the name of the VPC endpoint service.
```

Comment:
to keep compatible with `flexibleengine_vpcep_endpoint` resource, we can rename this argument to `service_name`.

Reply:
done